arangod/Scheduler/SupervisedScheduler.cpp

```diff
@@ -194,6 +194,8 @@ DECLARE_COUNTER(arangodb_scheduler_threads_started_total,
                 "Number of scheduler threads started");
 DECLARE_COUNTER(arangodb_scheduler_threads_stopped_total,
                 "Number of scheduler threads stopped");
+DECLARE_GAUGE(arangodb_scheduler_queue_memory, std::int64_t,
+              "Number of bytes allocated for tasks in the scheduler queue.");
```

Comment:
nit: this metric's description ends with a dot, the ones above don't.
I don't know what is correct, but it is inconsistent like this.
```suggestion
              "Number of bytes allocated for tasks in the scheduler queue");
```